packages/instant/src/components/ui/button.tsx

```diff
@@ -12,15 +15,26 @@ export interface ButtonProps {
     padding?: string;
     type?: string;
     isDisabled?: boolean;
-    onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+    href?: string;
+    onClick?: ButtonOnClickHandler;
```

Comment:
nit but would be nice for the type to ensure that we don't define both an `href` and an `onclick`

Reply:
I played around with this and couldn't find a solution I loved. Should `onClick` also be able to take in a `string`?

Reply:
`onClick` doing double duty of being able to take a string or function feels a little funky, i'm ok w/ this as-is